docs/pages/management/admin/troubleshooting.mdx

```diff
@@ -161,3 +161,29 @@ cluster, read our [Teleport Diagnostics](../../reference/metrics.mdx) guide.
 For additional sources of Teleport support, please see the
 [Teleport Support and Education Center](https://goteleport.com/support/).
 
+## Common Issues
+
+### `teleport.cluster.local`
+
+It is common to see references to `teleport.cluster.local` within logs and
+errors in Teleport. This is a special value that is used within Teleport for two
+purposes and seeing it within your logs is not necessarily an indication that
+anything is incorrect.
+
+Firstly, Teleport uses this value within certificates (as a DNS SAN) issued to
+the Auth and Proxy Service. Teleport clients can then use this value to
+validate the service's certificates during the TLS handshake regardless of the 
+service address as long as the client already has a copy of the cluster's 
+certificate authorities. This is important as there are often multiple different 
+ways that a client can connect to the Auth Service and these are not always via 
+the same address.
+
+Secondly, this value is used by clients as part of the URL when making gRPC or 
```

Comment:
The auth server actually ignores the Host header completely, FWIW 😬 

The important part (for TLS routing purposes) is using it as the SNI, so that the proxy (and all the TLS-aware and Teleport-aware hops before the proxy) can route the connection correctly.

Reply:
Yeah, I couldn't work out if there was a nice way of saying "The Teleport API completely ignores this" without it just causing more questions than answers.